sshfs.c

```diff
@@ -568,8 +568,8 @@ static inline void buf_init(struct buffer *buf, size_t size)
 		}
 	} else
 		buf->p = NULL;
-	buf->len = 0;
-	buf->size = size;
+		buf->len = 0;
+		buf->size = size;
```

Comment:
I think this is wrong, there are no curly braces on the else statement, these two lines run in both cases.

Reply:
Your alright i'll modify it.